### PR TITLE
Feat/added tooltips throughout the app

### PR DESCRIPTION
**Pull request by isha-dubey**
## Description

Feat/added tooltips throughout the app

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@environments/features/environment-explorer/layout/EnvironmentExplorer.svelte [hunk 1]
```diff
@@ -12,6 +12,7 @@
   import { TabularInput } from "@environments/common/components";
   import { WithButton } from "@environments/common/hoc";
   import { Input } from "@library/forms";
+    import { platform } from "@tauri-apps/plugin-os";
```
Comment (itsmdasifraza) [inline src/packages/@environments/features/environment-explorer/layout/EnvironmentExplorer.svelte:16]: is this in use?

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Pull request merged**